easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1212,6 +1229,22 @@ def all_dependencies(self):
 
         return self._all_dependencies
 
+    @property
+    def parallel(self):
```

Comment:
@Flamefire Introducing this makes sense, but it means that easyblocks will need to be updated accordingly, right?

Reply:
Correct. Should be pretty easy to search for `self.cfg['parallel']`. I mentioned the need to update the easyblocks in the description which we can only do after merging this PR.